### PR TITLE
[SMALLFIX] Improve Chinese document to make it more native

### PR DESCRIPTION
**Pull request by reshow**
change several words in docs/cn/Command-Line-Interface.md

### CODE REVIEW

File: docs/cn/Command-Line-Interface.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Usage: alluxio fs [generic options]
        ...
 ```
 
-对于用Alluxio URI（如`ls`， `mkdir`）作为参数的`fs`子命令来说，参数应该要么是完全的Alluxio URI `alluxio://<master-hostname>:<master-port>/<path>`，要么是没有头部信息的`/<path>`，以使用`conf/allluxio-site.properties`中设置的默认的主机名和端口。
+对于用Alluxio URI（如`ls`， `mkdir`）作为参数的`fs`子命令来说，参数应该要么是完整的 Alluxio URI `alluxio://<master-hostname>:<master-port>/<path>`，要么是省略了头部信息的`/<path>`，以使用`conf/allluxio-site.properties`中设置的默认的主机名和端口。
```
Comment (LuQQiu) [inline docs/cn/Command-Line-Interface.md]: Could me remove the extra space?
`参数应该要么是完整的 Alluxio URI` to `参数应该要么是完整的Alluxio URI`

Comment (reshow) [reply docs/cn/Command-Line-Interface.md]: Of course you can.

Comment (LuQQiu) [reply docs/cn/Command-Line-Interface.md]: @reshow sorry, that's my typo. I want to say "could you remove the extra space".
Could you remove the extra space and use the
`git add -u` + `git commit -m "remove space"` + `git push origin smallfix` commands to push one more commit to your branch `smallfix`.

Thanks!

Comment (reshow) [reply docs/cn/Command-Line-Interface.md]: Ok，I will do that as soon as possible.

Comment (reshow) [reply docs/cn/Command-Line-Interface.md]: I've done that.O(∩_∩)O

**Comment by LuQQiu** [comment]
jenkins, test this please

**Pull request merged**